### PR TITLE
Register Subdomain: eliask

### PR DESCRIPTION
**Pull request by eliaskyt**
Added the domain: eliask

### CODE REVIEW

**Review by MaskDuck — CHANGES_REQUESTED** [review]

File: domains/eliask.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+
+    {
+      "owner": {
+        "username": "eliaskyt",
+        "email": "elias@krueckeberg.eu"
+      },
+
+      "record": {
+        "CNAME": "eliaskrueckeberg.de"
```
[the remark this answers is not in the document]
Comment (eliaskyt) [reply domains/eliask.json:9]: I can access :)

[the remark this answers is not in the document]
Comment (akshayans) [reply domains/eliask.json:9]: I can, but no files are uploaded on the server it says. And, the site is not in English, so it still doesn't qualify. If your site is in a different language, add a translation.

[the remark this answers is not in the document]
Comment (eliaskyt) [reply domains/eliask.json:9]: Yes, im working on my site and want it to release on my is-a.dev Domain :)

[the remark this answers is not in the document]
Comment (WilliamDavidHarrison) [reply domains/eliask.json:9]: @eliaskyt Your website needs to completed before we can give you a domain.

**Comment by WilliamDavidHarrison** [comment]
PR been open for over 2 months with no changes, still a default webpage.

**Pull request closed without merging**